models/common_data_models/docs.md

```diff
@@ -0,0 +1,127 @@
+{% docs billing_type %}
+description
```

Comment:
Would you actually be able to update these to match [the following descriptions](https://github.com/fivetran-streamlit/standardized_billing_model/blob/fa8af19e0c720792af04c06ca119acc522d2b5fd/pages/billing_schema.py#L31-L64).

Reply:
In case you don't have access to the above repo:
```yml
schema = {
    "🔑 header_id": "ID of either the invoice or order.",
    "🔑 line_item_id": "ID of either the invoice line or order line item.",
    "line_item_index": "Numerical identifier of the line item within the header object.",
    "record_type": "Either 'header' or 'line_item' to differentiate if the record is originally from the line item table or was created to document information only available at the header level.",
    "created_at": "Date the line item entry was created.",
    "currency": "Determines the currency in which the transaction took place.",
    "header_status": "Indicates the status of the header. Eg. paid, voided, returned.",
    "product_id": "ID of the product associated with the line item.",
    "product_name": "Name of the product associated with the line item.",
    "product_type": "Type of the product (e.g., physical, digital).",
    "product_category": "Category to which the product belongs (e.g., electronics, clothing).",
    "quantity": "Quantity of the product in the line item.",
    "unit_amount": "Unit price of the product.",
    "discount_amount": "Amount of discount applied to the line item.",
    "tax_rate": "Tax rate applied to the line item.",
    "tax_amount": "Amount of tax applied to the line item.",
    "total_amount": "Total amount for the line item (including tax and discounts).",
    "payment_id": "ID of the payment associated with the line item.",
    "payment_method": "Method used for payment (e.g., credit card, PayPal).",
    "payment_at": "Date and time of the payment.",
    "fee_amount": "Any additional fees associated with the payment.",
    "refund_amount": "Amount refunded for the line item (if applicable).",
    "subscription_id": "ID of the subscription (if applicable).",
    "subscription_period_started_at": "Start date of the subscription period (if applicable).",
    "subscription_period_ended_at": "End date of the subscription period (if applicable).",
    "subscription_status": "Status of the subscription (if applicable).",
    "customer_id": "ID of the customer associated with the line item.",
    "customer_level": "Identifies if the customer is reported at the account or customer level.",
    "customer_name": "Name of the customer.",
    "customer_company": "Company name of the customer (if applicable).",
    "customer_email": "Email address of the customer.",
    "customer_city": "City of the customer's address.",
    "customer_country": "Country of the customer's address."
}
```

Reply:
@fivetran-joemarkiewicz A couple questions:
1. Should I customize the definitions for the package or is the idea to have a single set of definitions? For example, `header_id` is defined here as an invoice or order, but in recharge it is neither. It is charge_id, and the order_id is a different table and not used in this model. 
2. I had already created a `docs.md` that has definitions with the same name, like `customer_id` for example and causes a dbt error. Maybe should we hard code the definitions into the yml for the cdms to avoid this issue in this package and others? 
3. Just a note--`billing type`, `transaction_type`, and `payment_method_id` are missing from the linked list.

Reply:
@fivetran-catfritz let's let the definitions be similar to the standard definitions. However, to your point we should add some specifics for the platform (like the header id being charge_id). Yeah let's plan to have these be their own yml since I can see that conflicting with the base models.

Thanks for catching those, I can add them to the streamlit report.